ros2pkg/ros2pkg/resource/ament_python/setup.py.em

```diff
@@ -0,0 +1,28 @@
+from setuptools import setup
+
+package_name = '@project_name'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'@[for dep in dependencies]@ , '@dep'@[end for]@ ],
```

Comment:
I am not an expert on this, but I believe the `install_requires` are not for all dependencies, but only for the setuptools.

Reply:
Ooops!

 You are correct. Should be fixed now. I was able to make a project with a rclpy dependency and build and run it.